<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subplot_page</class>
 <widget class="QMainWindow" name="subplot_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="sub_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Build SubPlots</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="down_f">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>641</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="up_f">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>530</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 255, 189);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="display_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(166, 183, 255);</string>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QLabel" name="st_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Super Title:</string>
    </property>
   </widget>
   <widget class="QLabel" name="first_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>First line:</string>
    </property>
   </widget>
   <widget class="QLabel" name="second_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Second line:</string>
    </property>
   </widget>
   <widget class="QLabel" name="third_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Third line:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sixth_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Sixth line:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fifth_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Fifth line:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fourth_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Fourth line:</string>
    </property>
   </widget>
   <widget class="QLabel" name="q_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Quantity (item):</string>
    </property>
   </widget>
   <widget class="QLabel" name="color_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="marker_label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Marker Type:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="st_line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="first_line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="second_line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="fifth_line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="third_line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="fourth_line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sixth_line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="q_line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="color_box">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Chocolate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Purple</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="dq_box">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1:2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2:1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2:2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2:3</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="marker_box_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>o</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>D</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>v</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="dq_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Diagramm Quantity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="marker_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Marker Size:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="marker_box_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="marker_label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Marker Border Color:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="marker_box_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Brown</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Orange</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Olive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Indigo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Purple</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="marker_label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Marker Inside Color:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="marker_box_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Peru</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Salmon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SeaGreen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Silver</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tan</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
